app/views/handlers/thank_you.py

```diff
@@ -35,6 +38,7 @@ def __init__(self, schema):
         self.confirmation_email = (
             ConfirmationEmail(self.PAGE_TITLE)
             if self._schema.get_submission().get("confirmation_email")
+            and not email_limit_exceeded()
             else None
         )
```

Comment:
Not sure what peoples thoughts will be but now that `ConfirmationEmail` checks if the limit has been reached, the above code would check the if the limit has been reached twice. To avoid that, this could be something like:
```py
        self.confirmation_email = self._get_confirmation_email()

    def _get_confirmation_email(self):
        if not self._schema.get_submission().get("confirmation_email"):
            return None
        
        try:
            return ConfirmationEmail(self.PAGE_TITLE)
        except EmailLimitExceeded:
            return None
```

Reply:
Yeah I think this is probably better than checking twice, updated now